### ISSUE TITLE
Improve on WCAG "Understandable" category - a11y audit

### ISSUE THREAD

**Issue by xmedr** [Metro-Records/la-metro-councilmatic#1023]
Go through the following criteria for [WCAG's Understandable section](https://www.w3.org/TR/WCAG21/#understandable) and improve their issues to meet WCAG requirements:

### Level A
- 3.3.2 Labels or Instructions:
  - [x] The board report search bar has an `aria-describedby` that doesn’t lead to anything. Change that to an `aria-labelledby` and have it refer to the preceding text. Since that search gets used in a couple places, we may have to apply id's to each of those labels.
  - [x] The events and minutes searches have [input-groups that need aria-labels](https://getbootstrap.com/docs/5.0/forms/input-group/#basic-example)
  - [x] The person headshot form input needs an id to match the label

### Level AA
- 3.1.2 Language of Parts:
  - [x] [The Ver en Español links should have lang=”es”](https://www.w3.org/WAI/WCAG21/Understanding/language-of-parts.html#examples) (this [resource](https://www.w3.org/International/questions/qa-html-language-declarations) might help with links)

- 3.2.4 Consistent Identification
  - [x] The right chevron that takes you to a legislation in `_legislation_item.html` and `_search_result.html` should be accessibly labelled 

Connected to #1012

**Comment by xmedr** [comment]
Closed by #1058
